dowhy/causal_graph.py

```diff
@@ -401,10 +401,19 @@ def get_all_directed_paths(self, nodes1, nodes2):
         # convert the outputted generator into a list
         return [p for p in nx.all_simple_paths(self._graph, source=node1, target=node2)]
 
+    def has_directed_path(self, nodes1, nodes2):
+        """ Checks if there is any directed path between two sets of nodes.
+
+        Currently only supports singleton sets.
+        """
+        dpaths = self.get_all_directed_paths(nodes1, nodes2)
+        flag = True if len(dpaths)>0 else False
+        return flag
```

Comment:
Nit: This isn't necessary. How about `return len(dpaths) > 0` ?

Reply:
thanks, yes this is better.